tests/stress/fun_call_limit.py

```diff
@@ -0,0 +1,35 @@
+# Test the limit of the number of arguments to a function call.
+# This currently tests the case of *args after many positional args.
+
+
+def f(*args):
+    return len(args)
+
+
+def test(n):
+    s = "f(" + ",".join(str(i) for i in range(n)) + ", *('a', 'b'), 'c', 'd')"
```

Comment:
This might be a bit more legible using a format string (all of the quotes close to each other make me a bit cross-eyed).

```suggestion
    s = "f({}, *('a', 'b'), 'c', 'd')".format(",".join(str(i) for i in range(n)))
```

Reply:
updated, and changed the 1-char strings to numbers to make it easier to read